tests/main/static/task.yaml

```diff
@@ -0,0 +1,5 @@
+summary: Check that snapd can be built without cgo
+systems: [-ubuntu-core-16-64, -ubuntu-core-16-arm-64, -ubuntu-core-16-arm-32]
+execute: |
+    CGO_ENABLED=0 go build -o snapd.static github.com/snapcore/snapd/cmd/snapd
+    ldd snapd.static && exit 1 || true
```

Comment:
This is broken IMHO:

Replacing `ldd snapd.static` by `true` or `false` always results in the expression being true:

```
macbook-zyga:~ zyga$ true && false || true; echo $?
0
macbook-zyga:~ zyga$ false && false || true; echo $?
0
```

I suspect you need parentheses around this:

```
macbook-zyga:~ zyga$ false && (false || true); echo $?
1
```